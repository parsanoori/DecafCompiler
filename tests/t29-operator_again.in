/* define a b
define c d 
// define e f 
a
c
e
*/
a
c
e
