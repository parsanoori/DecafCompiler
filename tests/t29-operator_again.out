T_ID a
T_ID c
T_ID e